src/uwtools/drivers/ungrib.py

```diff
@@ -131,10 +141,49 @@ def _driver_name(self) -> str:
         """
         return STR.ungrib
 
+    @task
+    def _gribfile(self, link, infile):
```

Comment:
Suggestion: potentially move this method to conform to alphabetical ordering. It should be the first method call before `ungribfiles`.

Reply:
Nice catch. I did several iterations of what to call the variables and forgot to alphabetize them once I settled.